src/cmd1.cc

```diff
@@ -556,89 +554,6 @@ s16b tot_dam_aux(object_type *o_ptr, int tdam, monster_type *m_ptr,
 
 
 /*
- * Search for hidden things
- */
-void search(void)
```

Comment:
They will not. They can be tunneled through, but I will see if I can't remove them (or at least make them functionally equivalent to normal doors).